pkg/k8shandler/curation.go

```diff
@@ -114,9 +114,9 @@ func (cluster *ClusterLogging) createOrUpdateCuratorConfigMap() error {
 		"curator",
 		cluster.Namespace,
 		map[string]string{
-			"actions.yaml":  string(utils.GetFileContents("files/curator-actions.yaml")),
```

Comment:
if we do this then we can no longer use `make go-run`.

Reply:
I reverted this change